bin/varnishd/mgt/mgt_jail_unix.c

```diff
@@ -132,28 +143,25 @@ vju_init(char **args)
 			ARGV_ERR("Unix Jail: Must be root.\n");
 
 		for (;*args != NULL; args++) {
-			const char * const a_user = "user=";
-			const size_t l_user = strlen(a_user);
-			if (!strncmp(*args, a_user, l_user)) {
-				if (vju_getuid((*args) + l_user))
+			val = keyval(*args, "user=");
+			if (val != NULL) {
+				if (vju_getuid(val))
 					ARGV_ERR(
 					    "Unix jail: %s user not found.\n",
 					    (*args) + 5);
```

Comment:
Not ready to replace #4202 until you get rid of `(*args) + ...` occurrences.

Reply:
oops! thank you!

Reply:
That I had already overlooked in 4001ea251bac6924c2e86bac3d0f2c8d93c96bd8 🤦🏽‍♂️